src/daemon/rpc_command_executor.cpp

```diff
@@ -2284,6 +2294,12 @@ bool t_rpc_command_executor::print_sn(const std::vector<std::string> &args)
       }
     }
 
+    if (include_json)
+    {
+      tools::msg_writer() << "JSON Representation: " << std::endl << std::endl <<
```

Comment:
This adds some problems with parsing the output if you programatically wrote this to a log file since now they have to sanitise the file and remove this line from the output. It should just print out in raw json.